### DISCUSSION TITLE
Add a sports club

### DISCUSSION THREAD

**Discussion by MXC48-zz** [organicmaps/organicmaps#1718] [Q&A] [answered]
Hello, How to add a sports club on open street map with OrganicMap ?

**Comment by biodranik** [comment]
Looks like adding this type of POI is not supported yet in our built-in editor. Do you need it often? For now, you can add it directly on https://openstreetmap.org/

**Reply by MXC48-zz** [reply, marked as answer]
Thank you, yes I will start adding the sports clubs in my area.
